kubernetes/charts-linera/redeploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Helper script for re-installing helm charts locally.
+helm uninstall linera-core;
+sleep 0.5;
```

Comment:
Out of curiosity, why is the sleeping necessary?

Reply:
`helm uninstall` will change the desired state of the cluster but it takes some time for them to actually get spun down.